src/Utils/request/errorHandler.ts

```diff
@@ -97,3 +92,25 @@ function handlePydanticErrors(errors: PydanticError[]) {
     });
   });
 }
+
+function handleBadRequest(cause: any) {
+  const errs = cause?.errors;
+
+  if (isPydanticError(errs)) {
+    handlePydanticErrors(errs);
+    return;
+  }
+
+  if (cause && typeof cause === "object" && !Array.isArray(cause)) {
+    Object.values(cause).forEach((value) => {
+      if (Array.isArray(value)) {
+        value.forEach((err) => {
+          toast.error(`${err}`);
+        });
+      }
+    });
+    return;
+  }
```

Comment:
what i meant was, write checks and handler for this piece of logic alone, and not to move the entire thing as handleBadRequest. Code ain't readable this way.